app/commands/incvol.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Execute on incvol command
+ * @param {Discord.Message} msg - discord message
+ */
+const incvol = async (msg) => {
+    // check if user in the voice channel
+    const voiceChannel = msg.member.voice.channel;
+    if (!voiceChannel) {
+        return msg.channel.send(
+            `${msg.author.toString()} You need to be in a voice channel to decrease volume!`
+        );
+    }
+
+    let clientVoiceConnection = msg.guild.voice.connection;
+    if (!clientVoiceConnection) {
+        return msg.channel.send(`❌ Bot must be in voice channel`);
+    }
+
+    let percentage = 25 / 100; // if you want to use args just adjust the "25" to argument variable.
+
+    const serverQueue = musicQueue.get(msg.guild.id);
+    let volume = serverQueue.connection.dispatcher.volume + percentage;
```

Comment:
When increasing the volume, it can go beyond 100%. I think updating this line to this would fix the issue:
```javascript
let volume = Math.min(serverQueue.connection.dispatcher.volume + percentage,100);
```